imod/mf6/uzf.py

```diff
@@ -350,7 +358,7 @@ def _determine_vertical_connection(self, uzf_number):
         return uzf_number.shift(layer=-1, fill_value=0)
 
     def _package_data_to_sparse(self):
-        notnull = self.dataset["landflag"].values == 1
+        notnull = np.isfinite(self.dataset["landflag"].values)
```

Comment:
If this is the better condition, maybe just use `self.dataset["landflag"].notnull().to_numpy()`

Reply:
ok